engine/src/main/scala/cromwell/engine/workflow/workflowstore/WorkflowStoreCoordinatedWriteActor.scala

```diff
@@ -22,7 +22,7 @@ class WorkflowStoreCoordinatedWriteActor(workflowStore: WorkflowStore) extends A
   def run[A](future: Future[A]): Unit = {
     val result = Try(Await.result(future, Timeout)) match {
       case Success(s) => s
-      case f: Failure[_] => f
+      case f: Failure[_] => Status.Failure(f.exception)
```

Comment:
🤦‍♂️ 
And ToL: Maybe `case Failure(e) => Status.Failure(e)` is a tiny bit neater?

Reply:
Hehe, very good point. But this tiny fix has been a 💩-show. Will update you IRL if you care. Merging but PRs accepted. 😉